Makefile

```diff
@@ -107,7 +112,7 @@ ci-lint: golangci-lint
 	$(GOLANGCI_LINT) run --timeout 7m0s
 
 .PHONY: verify
-verify: vet ci-lint fmt-verify manifests generate
+verify: gomod-verify vet ci-lint fmt-verify manifests generate
```

Comment:
let's put the ones that modify the source at the end.

Reply:
I just wanted it to fail fast as `go mod tidy` is way cheaper than others. And a successful `go mod tidy` won't leave any rubbish on src code.

Reply:
sg. It will fail right away anyways.